src/main/webapp/app/quiz/participate/quiz.component.html

```diff
@@ -156,15 +161,18 @@ <h2 [ngSwitch]="mode">
             <div *ngIf="mode === 'solution'">
                 <span jhiTranslate="artemisApp.quizExercise.solution"></span>
             </div>
-            <button
-                id="submit-quiz"
-                *ngIf="!submission.submitted && !showingResult"
-                [disabled]="isSubmitting || waitingForQuizStart || remainingTimeSeconds < 0"
-                class="btn btn-success btn-lg"
-                jhiTranslate="entity.action.submit"
-                (click)="onSubmit()"
-            ></button>
-            <button *ngIf="submission.submitted && !showingResult" disabled class="btn btn-success btn-lg" jhiTranslate="artemisApp.quizExercise.submitted"></button>
+            <div class="submit-button">
+                <button
```

Comment:
In my opinion, this is a good candidate for using `jhi-button`. For example, button size in this case you will be able to set with `btnSize` input parameter.